### DISCUSSION TITLE
RSS Proxy 根据现有RSS进行额外调整

### DISCUSSION THREAD

**Discussion by NeverBehave** [DIYgod/RSSHub#6671] [Ideas]
参考: #6636 #6627

早期汇总: #122

转换需求被提出很多次, 目前已知类似服务有 https://siftrss.com , 但是做不到自定义(比如繁简转换)和自部署

如果有类似需求欢迎在这里留下用例, 或者提供更多数据点方便构建项目思路

**Comment by NeverBehave** [comment]
目前已知部分Github用户用于过滤官方Feed

**Comment by yefoenix** [comment]
是否做一个独立项目更合适？

**Reply by NeverBehave** [reply]
这里就是为了收集（RSS Proxy）作为独立项目的可能

**Comment by duhd1993** [comment]
提供一点用例:
- 部分源中含有http的图片链接，在https的rss阅读器中无法正常加载，可以通过cdn链接中转。https://github.com/DIYgod/RSSHub/pull/6790
- 媒体网站多个入口提供的源（如科技/最新），含有相同内容，需要去重。 https://github.com/DIYgod/RSSHub/issues/6636 。之前讨论说过可以客户端去重，但至少Inoreader只能在同一个源里面去重，并且还是付费功能。

**Comment by yefoenix** [comment]
想了一下可能的功能：

1. 格式转换，同时支持输出 rss2.0 与 atom 格式；
2. Feed 合并；
3. Feed 过滤；
4. Feed 翻译和繁简转换；
5. 抓取全文。
6. 将输入的任意 rss，重新输出为指定链接。
例如，我想把自己的 Twitter 内容通过 rsshub 建立的 feed，输出为 http://feeds.mydomain.com/twitter, 而 blog feed 输出为 http://feeds.mydomain.com/blog
7. 自定义 feed 的标题与描述，类似 FeedBurner 的 Title/Description Burner 功能；
8. 自定义 feed item 的内容，包括摘要输出（ Summary Burner ），分享增强（在内容尾部增加 微博、twitter 等服务的分享连接，类似 FeedBurner 的 FeedFlare 功能）；

感觉可以成为self-host的 Feedburner，需要有个后台吧。

**Reply by duhd1993** [reply]
我觉得还是做成类似rsshub这样无状态无后台的比较好...

**Reply by yefoenix** [reply]
没有后台的话也可以通过配置文件实现

**Reply by NeverBehave** [reply]
配置可以利用`query`进行动态调整，类似目前rsshub的操作

**Comment by NeverBehave** [comment]
https://github.com/DIYgod/RSSHub/issues/6785


可以拆分此功能

**Comment by yefoenix** [comment]
所以， RSSProxy 有进展了吗？

**Reply by NeverBehave** [reply]
有一点点头绪，后续会有一个草案出来

**Comment by cxplay** [comment]
支持(忘记搜索 discussions 就又提了个 [issue](https://github.com/DIYgod/RSSHub/issues/14430))
